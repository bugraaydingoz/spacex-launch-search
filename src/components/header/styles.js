import styled from 'styled-components'

export const HeaderContainer = styled.header`
  display: flex;
  flex-direction: column;
  align-items: center;
  margin: 60px 0 0 0;
  text-align: center;
`

export const Heading1 = styled.h1`
  display: flex;
  margin: 0 0 20px 0;
  font-weight: bold;
  font-size: 70px;

  @media (max-width: 768px) {
    font-size: 35px;
  }
`

export const Heading2 = styled.h2`
  display: flex;
  margin: 0;
  font-weight: normal;
  font-size: 24px;
`
