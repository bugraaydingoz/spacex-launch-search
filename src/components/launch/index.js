export { Launch } from './Launch'
export { LaunchList } from './LaunchList'
export { LaunchLoading } from './LaunchLoading'
