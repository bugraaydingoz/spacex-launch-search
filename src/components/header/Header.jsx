import React from 'react'

// Styles
import { HeaderContainer, Heading1, Heading2 } from './styles'

export function Header() {
  return (
    <HeaderContainer>
      <Heading1>SpaceX Monitoring</Heading1>
      <Heading2>
        An application that lists all SpaceX launches and launch details.
      </Heading2>
    </HeaderContainer>
  )
}
