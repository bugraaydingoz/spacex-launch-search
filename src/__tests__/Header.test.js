import React from 'react'
// import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'
import { Header } from '../components/header'

// test('has correct label', () => {
//   const label = 'length'
//   const { getByTestId } = render(
//     <Header name={label} value={{ min: 10, max: 20 }} />
//   )

//   expect(getByTestId('filter-label').innerHTML).toEqual(label)
// })

test('renders correctly', () => {
  const tree = renderer.create(<Header />).toJSON()
  expect(tree).toMatchSnapshot()
})
