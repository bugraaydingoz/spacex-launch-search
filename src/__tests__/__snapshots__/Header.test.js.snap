// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="sc-AxjAm bBVJHf"
>
  <h1
    className="sc-AxirZ HxwXi"
  >
    SpaceX Monitoring
  </h1>
  <h2
    className="sc-AxiKw fmMorb"
  >
    An application that lists all SpaceX launches and launch details.
  </h2>
</header>
`;
